spec.html

```diff
@@ -4055,17 +4045,20 @@ <h1>CopyDataBlockBytes ( _toBlock_, _toIndex_, _fromBlock_, _fromIndex_, _count_
       </emu-clause>
     </emu-clause>
 
-    <emu-clause id="sec-privatefield-record-specification-type">
-      <h1>The PrivateField Record Specification Type</h1>
-      <p>The PrivateField type is a Record used in the specification of private class fields.</p>
-      <p>Values of the PrivateField type are Record values whose fields are defined as by <emu-xref href="#table-privatefield-fields"></emu-xref>. Such values are referred to as <dfn>PrivateField Record</dfn>s.</p>
-      <emu-table id="table-privatefield-fields" caption="PrivateField Record Fields">
+    <emu-clause id="sec-privateelement-record-specification-type">
+      <h1>The PrivateElement Record Specification Type</h1>
+      <p>The PrivateElement type is a Record used in the specification of private class fields, methods, and accessors.</p>
+      <p>Values of the PrivateElement type are Record values whose fields are defined by <emu-xref href="#table-privateelement-fields"></emu-xref>. Such values are referred to as <dfn>PrivateElement Record</dfn>s.</p>
```

Comment:
TODO: call out that although these do not reuse the PropertyDescriptor type, these properties behave a lot like nonconfigurable properties (and nonwritable in the case of methods).